Gruntfile.js

```diff
@@ -136,23 +165,37 @@ module.exports = function(grunt) {
   grunt.initConfig({
     aws: grunt.file.readJSON('aws-keys.json'),
     aws_s3: awsS3Config,
-    cloudfront_invalidate: cloudfrontConfig,
+    // cloudfront_invalidate: cloudfrontConfig,
```

Comment:
:bulb: These are temporary comments?

Reply:
Yep, just ignoring CloudFront for now to simplify things.